rbd/rbd_test.go

```diff
@@ -1413,6 +1413,106 @@ func TestRemoveImage(t *testing.T) {
 	conn.Shutdown()
 }
 
+func TestCloneImage(t *testing.T) {
+	conn := radosConnect(t)
+
+	poolname := GetUUID()
+	err := conn.MakePool(poolname)
+	assert.NoError(t, err)
+
+	ioctx, err := conn.OpenIOContext(poolname)
+	require.NoError(t, err)
+
+	imageName := "parent1"
+	snapName := "snap1"
+	options := NewRbdImageOptions()
+	defer options.Destroy()
+	err = options.SetUint64(RbdImageOptionOrder, uint64(testImageOrder))
+	assert.NoError(t, err)
+	err = options.SetUint64(RbdImageOptionFeatures, 1)
+	assert.NoError(t, err)
+	err = CreateImage(ioctx, imageName, testImageSize, options)
+	assert.NoError(t, err)
+
+	image, err := OpenImage(ioctx, imageName, NoSnapshot)
+	assert.NoError(t, err)
+
+	snapshot, err := image.CreateSnapshot(snapName)
+	assert.NoError(t, err)
+	err = snapshot.Protect()
+	assert.NoError(t, err)
+
+	t.Run("cloneImage", func(t *testing.T) {
+		imageNames, err := GetImageNames(ioctx)
+		assert.NoError(t, err)
+		assert.Contains(t, imageNames, "parent1")
```

Comment:
instead of `"parent1"` use `imageName`, and add a variable for `"clone1"` too please. Or is there a specific reason you decided to not use variables for assert statements?